packages/editor/src/components/post-author/check.js

```diff
@@ -40,7 +40,10 @@ export default compose( [
 				false
 			),
 			postType: select( 'core/editor' ).getCurrentPostType(),
-			authors: select( 'core' ).getUsers( { who: 'authors' } ),
+			authors: select( 'core' ).getUsers( {
+				who: 'authors',
+				per_page: 100,
```

Comment:
Default in core is now 10? we need to ensure we always get enough to determine if the dropdown should be a combobox or select.